src/Nethermind/Nethermind.JsonRpc/Modules/Eth/GasPrice/GasPriceOracle.cs

```diff
@@ -31,7 +31,7 @@ public class GasPriceOracle : IGasPriceOracle
     {
         private readonly IBlockFinder _blockFinder;
         private readonly UInt256 _minGasPrice;
-        public UInt256 FallbackGasPrice => LastGasPrice ?? _minGasPrice;
+        public UInt256 FallbackGasPrice(in UInt256? baseFeePerGas = null) => LastGasPrice ?? GetMinimumGasPrice(baseFeePerGas ?? UInt256.Zero);
```

Comment:
In general, what is the reason for the fallback price to be public?
If it is public, why does it take baseFeePerGas instead of a block header?

Reply:
What I think is that -> when we have an interface then we can make it overly specific (BaseFee), or intuitively scoped (Header)

Reply:
Yeah I think it was made this way for tests only, will make private